### PR TITLE
Add tab in sidekiq web ui for bus info

### PR DESCRIPTION
**Pull request by verbad**
- copypasta from resque-bus for query construction
- add/register custom tab and routes for bus info and unsubscribe
- minimal styling of tables
- bump version

### CODE REVIEW

**Comment by bleonard** [comment]
Screenshot or it didn't happen.

**Comment by verbad** [comment]
Screen Shot 2019-07-10 at 11 24 54 AM

File: lib/sidekiq_bus/version.rb [hunk 1]
```diff
@@ -1,3 +1,3 @@
 module SidekiqBus
-  VERSION = "0.6.3"
+  VERSION = "0.6.4"
```
Comment (choubacha) [inline lib/sidekiq_bus/version.rb]: I'd rather add the changes to a changelog and release the gem versions in a separate commit/tag.

Comment (verbad) [reply lib/sidekiq_bus/version.rb]: fair enough

**Pull request merged**